SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealObjectRef.h

```diff
@@ -85,6 +69,9 @@ struct FUnrealObjectRef
 		return (*this != NULL_OBJECT_REF && *this != UNRESOLVED_OBJECT_REF);
 	}
 
+	UObject* ToObjectPtr(USpatialPackageMapClient* PackageMap, bool& bOutUnresolved) const;
```

Comment:
It would be nicer to make ToObjectPtr static as well with a signature:
static UObject* ToObjectPtr(const FUnrealObjectRef& ObjectRef, USpatialPackageMapClient* PackageMap, bool& bOutUnresolved);
so that it's consistent with FromObjectPtr. Maybe even move these two functions outside the class to some UnrealObjectRefUtilities namespace, but I'm not insisting on that.